packages/frontend/core/src/components/blocksuite/block-suite-header/present/use-present.ts

```diff
@@ -1,56 +1,13 @@
-import { useActiveBlocksuiteEditor } from '@affine/core/hooks/use-block-suite-editor';
-import type { EdgelessRootService } from '@blocksuite/blocks';
-import { useCallback, useEffect, useState } from 'react';
+import { EditorService } from '@affine/core/modules/editor';
+import { useLiveData, useService } from '@toeverything/infra';
+import { useCallback } from 'react';
 
 export const usePresent = () => {
```

Comment:
present is not actually the verb form of presentation. I guess it should be renamed to usePresentation instead